hw/xilinx/synth/constraints/Nexys-A7-50T-Master.xdc

```diff
@@ -120,8 +120,8 @@ set_property -dict {PACKAGE_PIN M17 IOSTANDARD LVCMOS33} [get_ports sys_reset_i]
 #set_property -dict { PACKAGE_PIN F6    IOSTANDARD LVCMOS33 } [get_ports { JC[2] }]; #IO_L19N_T3_VREF_35 Sch=jc[2]
 #set_property -dict { PACKAGE_PIN J2    IOSTANDARD LVCMOS33 } [get_ports { JC[3] }]; #IO_L22N_T3_35 Sch=jc[3]
 #set_property -dict { PACKAGE_PIN G6    IOSTANDARD LVCMOS33 } [get_ports { JC[4] }]; #IO_L19P_T3_35 Sch=jc[4]
-#set_property -dict { PACKAGE_PIN E7    IOSTANDARD LVCMOS33 } [get_ports { JC[7] }]; #IO_L6P_T0_35 Sch=jc[7]
-#set_property -dict { PACKAGE_PIN J3    IOSTANDARD LVCMOS33 } [get_ports { JC[8] }]; #IO_L22P_T3_35 Sch=jc[8]
+set_property -dict { PACKAGE_PIN E7    IOSTANDARD LVCMOS33 } [get_ports { uart_rx_i }]; #IO_L6P_T0_35 Sch=jc[7]

```

Comment:
We should do the same on the Nexys-A7-100T

Reply:
Indeed. Done

Reply:
Wait, we have to:
1. check the PMODs pins are the same
2. Actually test it on the board
3. Push the updated Nexys-A7-100T `xdc`

Reply:
50t and 100t only differs in the FPGA size. Pins voltage and mapping are exactly the same.
Nevertheless, An on-board test should be performed ofc.